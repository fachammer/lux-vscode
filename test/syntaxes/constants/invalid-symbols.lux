a#fa
a(
a[
a{