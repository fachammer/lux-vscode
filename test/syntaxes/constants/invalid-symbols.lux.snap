>a#fa
#^^^^ source.lux invalid.illegal.symbol.lux
>a(
#^^ source.lux invalid.illegal.symbol.lux
>a[
#^^ source.lux invalid.illegal.symbol.lux
>a{
#^^ source.lux invalid.illegal.symbol.lux